contrib/libdbusmenu-glib/patches/layout_fix_style.patch

```diff
@@ -0,0 +1,87 @@
+diff --git a/tests/test-glib-layout.h b/test-glib-layout.h
```

Comment:
are these patches necessary for build? (i.e. does the build fail without them in a way that cannot be fixed without introducing these patches?)

Reply:
yes, it fails during the checks without them ("error: use of GNU old-style field designator extension [-Werror,-Wgnu-designator]")

Reply:
that is because of Werror, which should be patched out